### ISSUE TITLE
Doc files rendered incorrectly

### ISSUE THREAD

**Issue by namvd2709** [nus-cs2103-AY1718S1/forum#54]
My doc files are rendered incorrectly. For example, when I run the `help` command, the following page appears: 
![screen shot 2017-09-27 at 12 27 54 am](https://user-images.githubusercontent.com/16438161/30893862-ed9ac382-a372-11e7-96f5-48c6b5df609c.png)

Running` ./gradlew processResources` returns a BUILD SUCCESSFUL with :asciidoctor UP-TO-DATE.
I have also tried to run `gradlew asciidoctor deployOfflineDocs` but the problem still persists.

**Comment by pyokagan** [comment]
Seems to be https://github.com/se-edu/addressbook-level4/issues/374

**Comment by Zhiyuan-Amos** [comment]
@namvd2709 yes, what senpai @pyokagan said above is right. If you scroll all the way down (https://github.com/se-edu/addressbook-level4/issues/374#issuecomment-314979746), you notice that this issue is fixed in JDK9. Don't worry about it; this isn't a major issue and you won't be penalised for it (am I right @damithc? Since this is an issue with the existing code base already.)

On a side note, if you are feeling ambitious, do feel free to update the project to use JDK 9 instead! I've just experimented with it and verified that it works (I suppose you are using a Mac right?). Hint on how to do so:
1. Update build.gradle
2. The compiler will complain because of some changes in Java 9 that caused certain packages to be no longer visible. As such, you have to update some command line parameters. I'll leave it to you to figure it out.
